README.md

```diff
@@ -1,6 +1,6 @@
 # AIOGram
 
-[![\[Telegram\] aiogram live](https://img.shields.io/badge/telegram-aiogram-blue.svg?style=flat-square)](https://t.me/aiogram_live)
+[![Financial Contributors on Open Collective](https://opencollective.com/aiogram/all/badge.svg?label=financial+contributors)](https://opencollective.com/aiogram) [![\[Telegram\] aiogram live](https://img.shields.io/badge/telegram-aiogram-blue.svg?style=flat-square)](https://t.me/aiogram_live)
```

Comment:
This badge should have the same style with other badges.
Can you add `style=flat-square` to badge src?

Reply:
I made this change